README.md

```diff
@@ -277,6 +284,11 @@ Once you have a development environment set up, you might find the following use
 | Run all tests | `pytest` | `pipenv run pytest` |
 | Build test coverage report | `pytest --cov-report html:cov_html --cov=./` | `pipenv run pytest --cov-report html:cov_html --cov=./` |
 
+If using make with pip, you should also be able to use the following:
+* `make` - ensure the development environment is up to date with specified dependencies
+* `make lint` - equivalent to `tools/lint-all`
+* `make test` - equivalent to running `make lint` and pytest
+
```

Comment:
We should add `make pytest` here too. I run the linters only after all my tests pass which could take a few iterations.

Reply:
I wasn't sure how much to duplicate the table - some things don't run via make, and others are equivalent except for testing venv. Prefixing `pytest` with `make` each time felt like maybe too much. Happy to include it though.

'fix' should likely depend on `venv` too.

Reply:
I am wary of `make` with `pytest` as well given that writing `make pytest` takes more time and hinders tab completion.

Reply:
You're meaning for pytest options?

Reply:
No, I meant in general. It is easier to do `pyte + <TAB>` than write `make pytest`.

Reply:
To clarify, we can still run `make pytest` and it will work (because of the way its listed in makefile) . Are we considering here whether to only mention it README or not?